app/views/pages/_home_responded_petitions.html.erb

```diff
@@ -7,7 +7,7 @@
         <h3><%= link_to petition.action, petition_path(petition), class: "threshold-petition-title" %></h3>
         <p class="intro">The government responded</p>
         <blockquote class="pull-quote"><%= simple_format(petition.government_response.summary) %></blockquote>
-        <p><%= link_to "Read the response in full", petition_path(petition) %></p>
+        <p><%= link_to "Read the response in full", petition_path(petition, show_response: "✓", anchor: 'response-threshold') %></p>
```

Comment:
Does this work on browsers that don't support unicode? I'm not sure of our browser coverage expectations.